### ISSUE TITLE
Example of a "Bucket curation" issue

### ISSUE THREAD

**Issue by zrlort** [dysonaj/sandbox#5]
Issues with the `bucket curation` tag will be used to document the process of curating the buckets.  For each topic, an issue should be created using the name of the topic as the title.  

The initial comment shall contain the following information:
  
- the initial (total) number of buckets,
- the initial number of topics related to the topic, and
- the initial list of buckets as an Excel attachment.

Any decision or conversation concerning the topic will be documented in the issue using the comments.

The last comment of the issue shall contain:
 - the final (total) number of buckets,
 - the final number of topics related to the topic, and
 - the final list of buckets as an Excel attachment.

Upon releasing the container, it will be indicated in the change log which (new) bucket topics have been included in the current version/release.

**Comment by dysonaj** [comment]
Diego mentioned a tag in the issue text. 
`bucket curation` or `oak tree` or `cloud patterns` could all be tags, theoretically. I've just put them as plain text between back-ticks to see what happens. Only `bucket curation` is actually a defined label.

**Comment by dysonaj** [comment]
Yep it's just Markdown, not linked to the label list. That's ok, just wanted to know.
